esp/src/src-react/layouts/DojoAdapter.tsx

```diff
@@ -89,8 +88,7 @@ export const DojoAdapter: React.FunctionComponent<DojoAdapterProps> = ({
             }
             widget = null;  //  Avoid race condition  ---
         };
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+    }, [delayProps, onWidgetMount, params, uid, widgetClass, widgetClassID]);
```

Comment:
This change causes an issue on the playground page, specifically the widgetClass & widgetClassID in the deps. The actual problem seems to be with line 107, the `widget?.resize();` Commenting out that line seems to fix it, and doesn't appear to change the behaviour of the graph. Also, the useEffect() above that (95-105) seem to be extraneous, I think due to the monitoring of the widget by this useEffect, so that could probably be removed as well.

Reply:
@jeclrsg I reverted